commit d0e209799a32903516017b84b5db18c5a6605779
Author: Bruno Haible <bruno@clisp.org>
Date:   Tue Jul 1 14:27:37 2025 +0200

    Fix an autoreconf failure in packages that only need AM_NLS.
    
    Rationale: https://lists.gnu.org/archive/html/bug-gettext/2025-06/msg00009.html
    
    * gettext-tools/m4/Makefile.am (aclocaldir, aclocal_DATA): New variables.
    * PACKAGING: Update.

diff --git a/PACKAGING b/PACKAGING
index 73fb94d..a520407 100644
--- a/PACKAGING
+++ b/PACKAGING
@@ -187,6 +187,7 @@ the following file list.
       $prefix/share/gettext/archive.cvs.tar.*   (only installed if --with-cvs specified)
       $prefix/share/gettext/disclaim-translations.txt
       $prefix/share/gettext-$VERSION/its/*
+      $prefix/share/aclocal/nls.m4
       $prefix/share/emacs/site-lisp/po-compat.el
       $prefix/share/emacs/site-lisp/po-compat.elc
       $prefix/share/emacs/site-lisp/po-mode.el
diff --git a/gettext-tools/m4/Makefile.am b/gettext-tools/m4/Makefile.am
index ba74da9..158c002 100644
--- a/gettext-tools/m4/Makefile.am
+++ b/gettext-tools/m4/Makefile.am
@@ -57,7 +57,17 @@ EXTRA_DIST = \
   locale-de.m4
 
 
-# List of all file names that we ever stored in $(aclocaldir).
+# Files installed for use by the old intltool.m4.
+# Listing nls.m4 is OK here, since it has essentially not changed over 20 years
+# and probably won't change in the future either.
+
+aclocaldir = @aclocaldir@
+aclocal_DATA = \
+  ../../gettext-runtime/m4/nls.m4
+
+
+# List of all file names that we ever stored in $(aclocaldir),
+# minus those that we still store in $(aclocaldir).
 OLD_M4_FILES = \
   build-to-host.m4 \
   codeset.m4 \
@@ -85,7 +95,6 @@ OLD_M4_FILES = \
   lock.m4 \
   longdouble.m4 \
   longlong.m4 \
-  nls.m4 \
   po.m4 \
   printf-posix.m4 \
   progtest.m4 \
